File: snippets/voice/conference/conference.py

from flask import Flask, request, redirect
from twilio import twiml
from twilio.twiml.voice_response import Conference, Dial, VoiceResponse
import requests 
import json

app = Flask(__name__)

@app.route("/conference", methods=['GET', 'POST'])
def conference():
   resp = VoiceResponse()
   dial = Dial()
   dial.conference('trainingconference')
   resp.append(dial)
   return str(resp)

File: snippets/voice/conference/conference.js
const express = require("express");
const app = express();


app.use('/conference', function(req, res) {
  const VoiceResponse = require('twilio').twiml.VoiceResponse;
  const response = new VoiceResponse();
  const dial = response.dial();
  dial.conference('Room 1234');
  res.send(response.toString());
})

// Make sure the port is the port ngrok is using.
http.createServer(app).listen(5000, () => {
  console.log("Express server listening on port 5000");
});
